packages/wallets/src/metamask/metamask.page.ts

```diff
@@ -61,6 +61,24 @@ export class MetamaskPage implements WalletPage {
     });
   }
 
+  async cancelAllTxInQueue() {
+    await test.step('Force close all transactions', async () => {
+      await test.step('Close all extension pages', async () => {
+        const pages = await this.browserContext.pages();
+        await Promise.all([
+          pages.map((page) => {
+            if (page.url().startsWith('chrome-extension://')) page.close();
+          }),
+        ]);
+      });
+      await this.initLocators();
+      await this.homePage.openWidgetPage();
```

Comment:
[Out of scope]. Could you rename `openWidgetPage();` -> `openExtensionPage()` since for now there is some missunderstanding in wording.

Reply:
sure!

Reply:
fixed